docs/support-bundle-guide.md

```diff
@@ -0,0 +1,247 @@
+
+| Collected Item             | Supported Collecting Method | Explanation                                                                                                                                                                                                                                                           |
+| -------------------------- | --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Antrea Agent Log           | A, O, SBC-N, SBC-E          | Log file of Antrea Agent                                                                                                                                                                                                                                              |
+| Antrea Controller Log      | C, O                        | Log file of Antrea Controller                                                                                                                                                                                                                                         |
+| iptables(Linux Only)       | A, O, SBC-N, SBC-E          | Result of running `ip6tables-save` and `iptable-save` with counters                                                                                                                                                                                                   |
+| OVS Ports                  | A, O, SBC-N, SBC-E          | Result of running `ovs-ofctl dump-ports-desc`                                                                                                                                                                                                                         |
+| NetworkPolicy Resources    | A, C, O, SBC-N, SBC-E       | YAML output of running `antctl get appliedtogroups`, `antctl get addressgroups`                                                                                                                                                                                       |
+| Heap Pprof                 | A, C, O, SBC-N, SBC-E       | Result of running [`pprof.WriteHeapProfile`](https://pkg.go.dev/runtime/pprof#WriteHeapProfile)                                                                                                                                                                       |
+| HNSResources(Windows Only) | A, O, SBC-N, SBC-E          | Result of running `Get-HnsNetwork` and `Get-HNSEndpoint`                                                                                                                                                                                                              |
```

Comment:
ditto

Reply:
Question:
is this correct `Get-HnsNetwork` and `Get-HNSEndpoint` ?
one is Hns*, another is HNS*

Reply:
Actually, Windows is insensitive on the character upper case or lower case, so both work well in Windows powershell. The standard command should be "HNS", updated.